projects/dsp-ui/src/lib/viewer/representation/upload-form/upload-form.component.html

```diff
@@ -0,0 +1,63 @@
+<div class="form-container">
+    <h2 class="mat-title">Add new resource: {{ resourceTyoe }}</h2>
+    <form [formGroup]="form">
+        <div dspDragDrop
+            *ngIf="isLoading || !file"
+            (click)="fileInput.click()"
+            (fileDropped)="addFile($event)"
+            class="dd-container">
+            <input hidden
+                type="file"
+                (change)="addFile($event)"
+                #fileInput />
+            <mat-icon *ngIf="!isLoading"
+                class="upload-icon">
+                cloud_upload
+            </mat-icon>
+            <dsp-progress-indicator *ngIf="isLoading"></dsp-progress-indicator>
+            <div class="title">{{ fromLabels.upload }}</div>
+            <div class="bottom-line">
+                {{ fromLabels.drag_drop }}
+            </div>
+        </div>
+
+        <ng-container *ngIf="!isLoading && file">
+            <div class="thumbnail">
+                <img src="{{ thumbnaillUrl }}"
+                    alt="thumbnail" />
+                <button mat-button
+                    title="delete file"
+                    (click)="deleteAttachment()">
+                    <mat-icon>close</mat-icon>
+                </button>
+            </div>
+
+            <div class="files-list">
+                <table>
+                    <thead>
+                        <tr>
+                            <th>Name</th>
+                            <th>Size</th>
+                            <th>Last Modified Date</th>
+                            <th>Delete</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>{{ file.name }}</td>
+                            <td>{{ convertBytes(file.size) }}</td>
+                            <td>{{ convertDate(file.lastModified) }}</td>
+                            <td>
+                                <button class="delete-file"
```

Comment:
Why not using `mat-icon-button` or `mat-button` here?

Reply:
`mat-button` has something on hover which I would need to override, but `mat-icon-button` works good. Foxed with f3b60c2.